brion/blueConfig.cpp

```diff
@@ -332,8 +339,7 @@ std::string BlueConfig::getCircuitTarget() const
 
 GIDSet BlueConfig::parseTarget( const std::string& target ) const
 {
-    return brion::Target::parse(
-        getTargets(), target.empty() ? _impl->getCircuitTarget() : target );
+    return brion::Target::parse( getTargets(), target );
```

Comment:
Did we agree on changing also this? This is affecting much more code than only Fivox, as the change in brain::Circuit does.